semgrep-core/src/parsing/pfff/ruby_to_generic.ml

```diff
@@ -177,6 +177,11 @@ and argument arg : G.argument =
   | Arg (Binop (Atom (_tk, AtomSimple id), (Op_ASSOC, _v2), v3)) ->
       let e3 = expr v3 in
       G.ArgKwd (id, e3)
+  (* Ruby allows surrounding single arguments with useless parenthesis *)
+  (* Ex: method(1,2,3) is equivalent to method((1),(2),(3))*)
+  | Arg (S (Block (l, [ S (Block (l2, [ e ], r2)) ], r))) ->
```

Comment:
You've only handled the case with 1 or 2 extra sets of parentheses. What if there were more? You'll need to deal with this recursively

Reply:
Meant to do that but it totally slipped my mind, will update this